### ISSUE TITLE
keyboard accessibility does not work with screen readers on

### ISSUE THREAD

**Issue by amolmathur** [Robdel12/DropKick#161]
Hi,

The keyboard navigation breaks in IE8 is screen reader is on. 
i tested with JAWS 14
